src/Interpreters/Context.cpp

```diff
@@ -1538,7 +1538,7 @@ BackgroundSchedulePool & Context::getMessageBrokerSchedulePool() const
         shared->message_broker_schedule_pool.emplace(
             settings.background_message_broker_schedule_pool_size,
             CurrentMetrics::BackgroundDistributedSchedulePoolTask,
-            "BgMsgBrkSchPool");
```

Comment:
But Len("BgMsgBrkSchPool") is 15

Reply:
the delay execution
thread's name (BgMsgBrkSchPool/D) is more than 15 chars.